cli-output/src/display.rs

```diff
@@ -463,6 +465,21 @@ fn write_balances<W: io::Write>(
     Ok(())
 }
 
+fn write_return_data<W: io::Write>(
+    w: &mut W,
+    return_data: Option<&TransactionReturnData>,
+    prefix: &str,
+) -> io::Result<()> {
+    if let Some(return_data) = return_data {
+        if !return_data.data.is_empty() {
+            writeln!(w, "{}Return Data:", prefix)?;
+            writeln!(w, "{}  Program: {}", prefix, return_data.program_id)?;
+            writeln!(w, "{}  Data: {:?}", prefix, return_data.data)?;
```

Comment:
This could be pretty big, couldn't it? Should we encode this? Or use pretty_hex, like we do for `solana account`?

Reply:
Unfortunately, it won't maintain the prefix unless we copy the code at https://github.com/wolandr/pretty-hex/blob/8f733950a964dbeb7451536ebcd1daa93cd3be50/src/pretty_hex.rs#L110 to add it in.  You can see the change in 4455f92cd4. What do you think?

Reply:
Personally, I don't think it's worth the duplication to add in the prefix. If anyone is parsing this programmatically, they're using json output, not counting white space. So I like it fine the way you've written it.